src/help/components/landing/styles/help-quick-search.module.scss

```diff
@@ -0,0 +1,42 @@
+@import 'franklin-sites/src/styles/colours';
+
+.help-autocomplete {
+  &__result-list {
+    max-height: 30rem;
+    overflow-y: auto;
+  }
+
+  &__all-link {
+    text-align: right;
+  }
+
+  :global(.search-input__suffix) {
```

Comment:
I guess they have to be `:global` to style Franklin components, but it looks like `.search-input__suffix` is also defined globally here https://github.com/ebi-uniprot/uniprot-website/pull/438/files?file-filters%5B%5D=.scss&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-3243e9ae200e982d2c3fe06ca36dd4fd9f387b7036bef39d8ba05f4292c9174bR21

Reply:
As discussed in slack it's OK to duplicate this in two separate places